src/app/clusters/joint-fabric-administrator-server/joint-fabric-administrator-server.cpp

```diff
@@ -123,10 +152,18 @@ bool emberAfJointFabricAdministratorClusterAddICACCallback(
 
     ChipLogProgress(Zcl, "JointFabricAdministrator: Received a AddICAC command");
 
+    auto nonDefaultStatus = Status::Success;
+
+    auto & failSafeContext = Server::GetInstance().GetFailSafeContext();
+    VerifyOrExit(failSafeContext.IsFailSafeArmed(commandObj->GetAccessingFabricIndex()),
+                 nonDefaultStatus = Status::FailsafeRequired);
```

Comment:
Where is the "received over CASE" check from the spec?

Reply:
See spec fix: now both the ICACCSRRequest and AddICAC are required to be received over CASE.

Reply:
Yes, that's fine for the spec, but my point was: the code is not doing that check, right?